hydromt/data_catalog.py

```diff
@@ -1229,10 +1229,15 @@ def export_data(
                                 variables=source_vars.get(key, None),
                                 bbox=bbox,
                                 time_tuple=time_tuple,
+                                handle_nodata=NoDataStrategy.RAISE,
```

Comment:
See my comment in dataframe. If NoDataStrategy is IGNORE, None is returned and I am not sure that case is handled well in this function?
Oh is this actually why you spcifically pass here RAISE and use handle_nodata and continue in case of IGNORE?
Then it's actually good, ignore my comment :)

Reply:
That is indeed why :)